feature-libs/quote/components/details/overview/quote-details-overview.component.html

```diff
@@ -1,78 +1,69 @@
+  <div class="cx-header-container">
+    <h3 class="cx-header">
       {{ 'quote.commons.id' | cxTranslate }}:
       {{ quoteDetails.code }}
     </h3>
-    <h3 class="status">
-      {{ 'quote.commons.status' | cxTranslate }}
+    <h3 class="cx-status">
+      {{ 'quote.commons.status' | cxTranslate }}:
       {{ 'quote.states.' + quoteDetails.state | cxTranslate }}
     </h3>
   </div>
-  <div class="cx-quote-overview">
-    <div class="container">
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(quoteDetails.code, 'quote.details.code') | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quoteDetails.creationTime | cxDate,
-              'quote.details.created'
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quoteDetails.updatedTime | cxDate,
-              'quote.details.lastUpdated'
-            ) | async
-          "
-        ></cx-card>
-      </div>
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(
-              getTotalPrice(quoteDetails),
-              getTotalPriceDescription(quoteDetails)
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          class="cx-card-description"
-          [content]="
-            getCardContent(
-              quoteDetails.description,
-              'quote.details.description'
-            ) | async
-          "
-          [truncateText]="true"
-          [charactersLimit]="30"
-        ></cx-card>
-      </div>
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(
-              'quote.states.' + quoteDetails.state | cxTranslate,
-              'quote.commons.status'
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quoteDetails.expirationTime | cxDate,
-              'quote.details.expiryDate'
-            ) | async
-          "
-        ></cx-card>
-      </div>
+  <div class="cx-container">
+    <div class="cx-summary-card">
+      <ng-container *ngIf="!editMode">
+        <div class="cx-content">
+          <cx-card
+            [content]="
+              getQuoteInformation(quoteDetails.name, quoteDetails.description)
+                | async
+            "
+          ></cx-card>
+        </div>
+        <div
+          *ngIf="isQuoteInformationEditable(quoteDetails)"
+          class="cx-edit-step"
+        >
+          <button
+            class="btn btn-link cx-action-link"
+            (click)="toggleEditMode()"
+          >
+            <cx-icon [type]="iconTypes.PENCIL"></cx-icon>
+          </button>
+        </div>
+      </ng-container>
+      <ng-container *ngIf="editMode">
+        <div class="cx-content">
```

Comment:
you could merge the *ngIf into the div to save a hierarchy level

Reply:
@Uli-Tiger, if I follow your proposal after clicking the pencil, the the card look the following way in edit mode:
![image](https://github.com/SAP/spartacus/assets/61147963/83c0eca1-0316-4151-9b5a-70ee66dc6693)

Reply:
I think  it is not the way how it should look like

Reply:
agree - you must have changed something else - for me it looks like this
![image](https://github.com/SAP/spartacus/assets/53090888/2a339ff9-adda-42ed-afb9-0dd5f98896e2)